src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -160,10 +171,9 @@ int64_t decode_with_past(ov::InferRequest& decoder_with_past,
                          const std::vector<int64_t>& generated_tokens) {
     // FIXME: Avoid this cast to i32. Why it's not i64 precision in model?
     decoder_with_past.get_tensor("input_ids").data<int32_t>()[0] = static_cast<int32_t>(input_id);
```

Comment:
Since optimum creates i64 dtype for input_ids, do we still need this cast? This cast was initially required to support NPU-friendly models.

Reply:
I believe it's attempt to align model generated on-the-fly to NPU-friendly one. Perhaps just leave it as `i64` is fine